custom-routes-vis/src/main/webapp/script.js

```diff
@@ -73,6 +77,40 @@ function updateCoordinates(lat, lng, containerId) {
       "Latitude: " + lat + "<br>Longitude: " + lng + "<br>";
 }
 
+/**
+ * Updates the deep linking url to various test apps on the page.
+ * TODO: Update Route Token if we get access to Routes Preferred API.
+ */
+function updateDeepLinkingUrl() {
+  if (document.getElementById('origin-coordinates').innerHTML != '' && 
+      document.getElementById('destination-coordinates').innerHTML != '') {
+    document.getElementById(IOS_URL_ID).innerHTML =
+        '<a href=navsdkdemo://advanced?originLat=' + originDestinationMarkers[0].position.lat()
+        + '&originLng=' + originDestinationMarkers[0].position.lng() + '&destLat=' 
+        + originDestinationMarkers[1].position.lat() + '&destLng=' 
```

Comment:
Consider creating variables at the start of this function for originDestinationMarkers[0].position and originDestinationMarkers[1].position since you reuse them here and below.